rules/the-best-dependency-injection-container/rule.md

```diff
@@ -24,14 +24,12 @@ redirects:
 
 ---
 
-You can waste days evaluating IOC containers. The top ones are quite similar. There is not much in this, but the best ones are StructureMap and AutoFac. At SSW we use Autofac on most projects.
+IoC containers are powerful tools that implement the IoC principle and automate the process of dependency resolution and object instantiation. They serve as central repositories of services and manage the lifecycle of objects. Some popular IoC containers in the .NET ecosystem include StructureMap, Autofac, Castle Winsdor, Ninject, Unity. At SSW we recommend using [.NET built-in Dependency Injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection) as default. Read more on [Dependency injection in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-8.0).
```

Comment:
Can you move the options to a separate list and make it clear which one is recommended, which ones are neutral, and which ones are NOT recommended?

Reply:
Similar to the options in: https://www.ssw.com.au/rules/use-mass-transit/

Reply:
Done.